packages/framework/request-handler/src/test/requestHandlers.spec.ts

```diff
@@ -92,7 +92,7 @@ describe("RequestParser", () => {
             const requestParser = RequestParser.create({ url: "/objectId/route", headers: { wait: true } });
             const response = await rootDataStoreRequestHandler(requestParser, runtime);
             assert.equal(response?.status, 200);
-            assert.equal(response.value.route, "/route");
+            assert.equal(response?.value.route, "/route");
```

Comment:
I think this is an unrelated change?

Reply:
A little bit,  but I keep on hitting compile error locally. So I just fixed that.  Not sure why it does fail in the CI